.github/workflows/sample.yml

```diff
@@ -0,0 +1,26 @@
+name: sample
```

Comment:
A more descriptive name? (same for filename)

Reply:
Yep, done.